crawl-ref/source/ng-setup.cc

```diff
@@ -509,6 +509,9 @@ static void _setup_generic(const newgame_def& ng)
     // Must be after _give_basic_knowledge
     add_held_books_to_library();
 
+    if (you.char_class = JOB_WANDERER)
```

Comment:
This should be an "==" rather than an "=".

Reply:
Oh whoops, always need a second pair of eyes :)